src/components/Board/components/Column/index.js

```diff
@@ -18,8 +18,7 @@ export const StyledColumn = styled.div`
 `
 
 const DroppableColumn = withDroppable(styled.div`
-  height: inherit;
-  min-height: inherit;
+  min-height: 28px;
```

Comment:
@Sabrinovsky can you take a look at this? You added these `inherit` for some reason I don't remember.

Reply:
@MatheusPoliCamilo If you set the DroppableColumn fixed, you wont be able to drop a card at the end of the column when the column has a fixed size.

Check this issue: https://github.com/lourenci/react-kanban/issues/300

Basically, you are able to create e column bigger than the default. If you set the column to be somethnig like 500px, the droppable area will still remain 28px. That way, you'll be able to drap cards on these 28px.

You can try inherit or setting it to 100%.

Reply:
@Sabrinovsky If I do what you suggested, the problem of the column size without cards getting bigger than another without cards during the renaming persists.

Do you imagine out any other resolution to the problem, without bringing this issue #300 back again?